docs/shared/recipes/add-stack/add-nuxt.md

```diff
@@ -0,0 +1,139 @@
+# Add an Nuxt Project
+
+The code for this example is available on GitHub:
+
+{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/nuxt-integrated" /%}
+
+**Supported Features**
+
+Because we are not using an Nx plugin for Nuxt, there are few items we'll have to configure manually. We'll have to configure our own build system. There are no pre-created Nuxt-specific code generators. And we'll have to take care of updating any framework dependencies as needed.
+
+{% pill url="/core-features/run-tasks" %}✅ Run Tasks{% /pill %}
+{% pill url="/core-features/cache-task-results" %}✅ Cache Task Results{% /pill %}
+{% pill url="/core-features/remote-cache" %}✅ Remote Caching{% /pill %}
+{% pill url="/core-features/explore-graph" %}✅ Explore the Graph{% /pill %}
+{% pill url="/core-features/distribute-task-execution" %}✅ Distribute Task Execution{% /pill %}
+{% pill url="/core-features/integrate-with-editors" %}✅ Integrate with Editors{% /pill %}
+{% pill url="/core-features/automate-updating-dependencies" %}✅ Automate Updating Nx{% /pill %}
+{% pill url="/recipes/enforce-module-boundaries" %}✅ Enforce Project Boundaries{% /pill %}
+{% pill url="/core-features/plugin-features/use-task-executors" %}🚫 Use Task Executors{% /pill %}
+{% pill url="/core-features/plugin-features/use-code-generators" %}🚫 Use Code Generators{% /pill %}
+{% pill url="/core-features/automate-updating-dependencies" %}🚫 Automate Updating Framework Dependencies{% /pill %}
+
+## Create Nuxt App
+
+Using `nuxi init`, `cd` to the directory above where you want the app folder to live and run
+
+```shell
+❯ npx nuxi@latest init <app-name>
+Nuxi 3.6.5
+✨ Nuxt project is created with v3 template.
+```
+
+Tell Nx how to cache the build with the `nx` key in the `package.json`.
+You'll also want to move the dependencies/devDependencies from the project level `package.json` to the top level `package.json` to help maintain a [single version policy](/concepts/more-concepts/dependency-management#single-version-policy).
+
+```json {% fileName="apps/<app-name>/package.json" %}
+{
+  "name": "<app-name>",
+  "scripts": {/***/}
+  "nx": {
+    "targets": {
+      "build": {
+        "outputs": ["{projectRoot}/.output", "{projectRoot}/.nuxt"]
+      }
+    }
+  }
+}
+```
+
+Make sure to ignore the `.output` and `.nuxt` folders from git and the Nx cache inputs.
```

Comment:
If you add `.output` and `.nuxt` to `.gitignore`, then you don't need to exclude them from inputs.  Nx will automatically ignore anything that git ignores.

Reply:
I just changed it to say move it to the top level dir so nx ignores it.